docs/maps/import-geospatial-data.asciidoc

```diff
@@ -58,6 +58,55 @@ On the {kib} home page, you can upload a file and import it into an {es} index w
 . Use the file chooser to select a GeoJSON file.
 . Click *Import file*.
 
+[discrete]
+=== Draw features in a map
+
+Upload features into {es} by drawing lines, polygons, circles, bounding boxes, and points in a map.
+
+To create a new index for drawing:
+
+. <<maps-create, Create a map>>.
+. Click *Add layer*.
+. Select *Create index*.
+. Set *Index name*.
+. Click *Create index*.
+
+To open an existing index for drawing:
```

Comment:
```suggestion
To open a previously created drawing index for editing:
```

Maybe not a great suggestion. But I do think we might need to find a way to distinguish between a drawing index and a normal index (like `kibana_sample_data_flights`) where a normal index cannot be edited. Or maybe we add something to line 82 about this.

Reply:
why limit editing to only indices created by "create index"? I think users should be able to edit features on any index that they have write permissions on. Really, "create index" is just a short cut for creating an index but there is nothing special about that index.

Reply:
Nathan and I talked offline and I was incorrect. It is possible to edit an existing index that was not created as a drawing layer.